caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -358,75 +370,46 @@ def AddBookkeepingOperators(model):
-
-# In[ ]:
-
-
-with open(os.path.join(root_folder, "train_net.pbtxt"), 'w') as fid:
-    fid.write(str(train_model.net.Proto()))
-with open(os.path.join(root_folder, "train_init_net.pbtxt"), 'w') as fid:
-    fid.write(str(train_model.param_init_net.Proto()))
-with open(os.path.join(root_folder, "test_net.pbtxt"), 'w') as fid:
-    fid.write(str(test_model.net.Proto()))
-with open(os.path.join(root_folder, "test_init_net.pbtxt"), 'w') as fid:
-    fid.write(str(test_model.param_init_net.Proto()))
-with open(os.path.join(root_folder, "deploy_net.pbtxt"), 'w') as fid:
-    fid.write(str(deploy_model.net.Proto()))
-print("Protocol buffers files have been created in your root folder: " + root_folder)
-
-
-# Next we will run the training procedure. We will drive all the computation in Python here, however you can also write a plan out to disk so that you can completely train stuff in C++.  We'll leave discussion on that route for another tutorial.
-# 
-# Please note that this process will take a while to run. Keep an eye on the asterisk (In [\*]) or other IPython indicators that the code block is still running.
-# 
-# First we must initialize the network with:
+# Next we will run the training procedure. Please note that this process will take a while to run. Keep an eye on the asterisk (In [\*]) or other IPython indicators that the code block is still running.
 # 
-#     workspace.RunNetOnce(train_model.param_init_net)
-#     
-# Since we are going to run the main network multiple times, we first create the network which puts the actual network generated from the protobuf into the workspace.
-# 
-#     workspace.CreateNet(train_model.net)
-# 
-# We will set the number of iterations that we'll run the network to 200 and create two numpy arrays to record the accuracy and loss for each iteration.
-# 
-#     total_iters = 200
-#     accuracy = np.zeros(total_iters)
-#     loss = np.zeros(total_iters)
-# 
-# With the network and tracking of accuracy and loss setup we can now loop the 200 interations calling `workspace.RunNet` and passing the name of the network `train_model.net.Proto().name`. On each iteration we calculate the accuracy and loss with `workspace.FetchBlob('accuracy')` and `workspace.FetchBlob('loss')`.
-# 
-#     for i in range(total_iters):
-#         workspace.RunNet(train_model.net.Proto().name)
-#         accuracy[i] = workspace.FetchBlob('accuracy')
-#         loss[i] = workspace.FetchBlob('loss')
+# We perform training by just executing our network many times in a row. Note how during this process we can fetch values of any blobs in the workspace. This allows us to build training plots. 
 # 
-# Finally, we can plot the results using `pyplot`.
+# If using MLP mode, with currently used architecture, model accuray highly depends on the initial initialization. If you see model being only about 50% accurate, you could try re-running the notebook. Which will make it start from another randomly initialized values for parameters.
```

Comment:
When using MLP, model accuracy greatly depends on the random initialization of parameters. If your model is staying at about 50% accurate, re-run the notebook, which will start from another random seed and initialize the parameters again.